react-app/src/library/services/vendor-implementations/plantronics/plantronics.ts

```diff
@@ -149,10 +191,10 @@ export default class PlantronicsService extends VendorImplementation {
       return [];
     }
 
-    if (!Array.isArray(result.Calls)) {
+    if (!Array.isArray(result?.Result?.Calls)) {
       return [];
     }
-    return result.Calls.filter(call => call.Source === this.pluginName);
+    return result?.Result?.Calls.filter(call => call.Source === this.pluginName);
   }
 
   async getCallEvents() {
```

Comment:
I can't comment where I want to, but on [line 219](https://github.com/purecloudlabs/softphone-vendor-headsets/pull/15/files#diff-e4a3c152b51cd8afb64bedffa38fa5615339dde0fc7a908226b689716f4a42d9R219) it looks like some fairly important call controls are guarded by `this.logHeadsetEvents`. Is that intended?